cypress/e2e/buy-shirt.cy.ts

```diff
@@ -1,22 +1,33 @@
-describe("Buy a t-shirt", () => {
-  it("then the t-shirt should be bought", () => {
-    cy.visit("http://automationpractice.com/");
-    cy.get("#block_top_menu > ul > li:nth-child(3) > a").click();
-    cy.get("#center_column a.button.ajax_add_to_cart_button.btn.btn-default").click();
-    cy.get("[style*=\"display: block;\"] .button-container > a").click();
-    cy.get(".cart_navigation span").click();
 
-    cy.get("#email").type("aperdomobo@gmail.com");
-    cy.get("#passwd").type("WorkshopProtractor");
+import {AddressStepPage, MenuContentPage} from "../page/index";
+import {ProducsList, ShippingStepPage} from "../page/index";
+import { ShoppingCart, PaymentStepPage } from "../page/index";
+import { LoginPage } from "../page/index";
+
+const menuContentPage = new MenuContentPage();
+const productsListPage = new ProducsList();
+const shoppingCartPage = new ShoppingCart();
+const loginPage = new LoginPage();
+const addressStepPage = new AddressStepPage();
+const shippingStepPage = new ShippingStepPage();
+const paymentStepPage = new PaymentStepPage();
 
-    // Debes completar la prueba ...
+describe("Buy a t-shirt", () => {
+  it("then should be bought a t-shirt", () => {
+    menuContentPage.visitMenuContentPage();
+    menuContentPage.goToTShirtMenu();
+    productsListPage.clickAddToCartBtn();
+    shoppingCartPage.clickProceedBtn();
+    shoppingCartPage.clickSecondCheckout();
+    loginPage.enterCredentials();
```

Comment:
These credential should be passed as parameters, is better in case we need flexibility